ynput/OpenPype
# Applications: Environment variables order

## Changelog Description
Changed order of set environment variables. First are set context environment variables and then project environment overrides. Also asset and task environemnt variables are optional.

## Additional info
This change gives option to use context environment variables in project environment variables.

## Testing notes:
1. Add custom env variable to project environments with context variable (e.g. `"MY_ENV": "Test: {AVALON_PROJECT}")
2. Launch an application (e.g. nuke, maya)
3. Check if the env is there and has filled context variable
